Scripts/operations.py

```diff
@@ -322,6 +328,50 @@ def check_duplicates(current, config, miscData, allCommentsDict, videoID):
       scannedCount +=1
       print(f" Analyzing For Duplicates: [ {scannedCount/authorCount*100:.2f}% ]   (Can be disabled & customized in config)".ljust(75, " "), end="\r")
 
+  # Look for multi-author duplicates
+  dupeCommentsDict = []
+
+  for commentDict in flatCommentsDict:
+
+    # Store if we don't have anything to iterate
+    if (len(dupeCommentsDict) == 0):
+      dupeCommentsDict.append({commentDict['commentText']: [commentDict]})
+    else:
+      matched = False
+      for i,x in enumerate(dupeCommentsDict):
+        key = next(iter(x))          
+        if ratio(key, commentDict['commentText']) > levenshtein:
+
+          # Don't store comments by the same author (delegates responsibility to main dupe checker... we're looking for multiple authors)
```

Comment:
There's potentially an edge case where someone has repeated a comment less than minDuplicates and hence this omits a dupe comment from being deleted